system/prometheus-vmware/templates/podmonitors/vrops-exporter.yaml

```diff
@@ -0,0 +1,53 @@
+{{- if .Values.vropsExporter.enabled }}
```

Comment:
I wouldn't go with CameCase here, maybe `vrops-exporter`? That would fit quite better to its peer `kube-state-metrics-exporter`

Or is it for better searchability?